tooling/cli/src/build.rs

```diff
@@ -272,22 +271,17 @@ pub fn setup(
   options: &mut Options,
   mobile: bool,
 ) -> Result<()> {
-  let (merge_config, merge_config_path) = resolve_merge_config(&options.config)?;
-  options.config = merge_config;
-
-  let config = get_config(target, options.config.as_deref())?;
+  let config = get_config(target, options.config.as_ref().map(|c| &c.0))?;
```

Comment:
can't we just config as an argument to the `setup` function?

Reply:
we do not do this overall in the codebase, get_config is cached so it's cheap if it's already been resolved, otherwise we need to pass the config around all codebase

Reply:
yeah, but config is parsed right before calling `setup` so it is better to just pass the config to it, than getting from cache again

Reply:
you're right it wasn't that complicated in the end.. thanks